frontend/javascripts/oxalis/model/sagas/save_saga.js

```diff
@@ -555,20 +606,14 @@ function* applyAndGetRevertingVolumeBatch(
     // Prepare a snapshot of the bucket's current data so that it can be
     // saved in an VolumeUndoState.
     let bucketData = null;
+    const currentPendingOperations = bucket.pendingOperations.slice();
     if (bucket.hasData()) {
       // The bucket's data is currently available.
       bucketData = bucket.getData();
       if (compressedBackendData != null) {
         // If the backend data for the bucket has been fetched in the meantime,
-        // we can first merge the data with the current data and then add this to the undo batch.
```

Comment:
I think this code was simply wrong and I also managed to produce a failing test which now succeeds with the new change.